bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerMetadata.java

```diff
@@ -103,7 +103,7 @@ public LedgerMetadata(int ensembleSize, int writeQuorumSize, int ackQuorumSize,
         this.lastEntryId = LedgerHandle.INVALID_ENTRY_ID;
         this.metadataFormatVersion = CURRENT_METADATA_FORMAT_VERSION;
 
-        this.digestType = digestType.equals(BookKeeper.DigestType.MAC) ?
+        this.digestType = BookKeeper.DigestType.MAC.equals(BookKeeper.DigestType.MAC) ?
```

Comment:
what does this change do?

Reply:
it is a stupid mistake, I don't know how it was possible